web/src/util/serialization/activities.test.js

```diff
@@ -186,7 +186,7 @@ describe('APD activity serializer', () => {
       expect(fromAPI(activityAPI, ['1947', '1949'])).toMatchObject(
         expect.objectContaining({
           id: 998,
-          key: expect.stringMatching(/[a-f0-9]{8}/),
+          key: expect.stringMatching(/^[a-f0-9]{8}$/),
```

Comment:
Tightens up these regexes.  The entire string should be matched, hence adding the start-of-line and end-of-line markers to the check.  There was one set of these that was only looking for 6 characters and without the start/end markers, it passed just fine.  But it was wrong.

![sad ant](https://media.giphy.com/media/dM8fBhhhAmbaU/giphy.gif)

Reply:
![judging you](https://media1.tenor.com/images/11d3354de6901e48d4f93097270a680e/tenor.gif?itemid=5169083)